types/jasmine/index.d.ts

```diff
@@ -718,13 +718,13 @@ declare module "jasmine" {
         addSpecFile(filePath: string): void;
         addSpecFiles(files: string[]): void;
         configureDefaultReporter(options: any, ...args: any[]): void;
-        execute(files: any, filterString: any): any;
+        execute(files?: string[], filterString?: string): any;
```

Comment:
as the documentation states that you can call it either without any argument or with both `files` and `filterString` I'd propose to add a overload with the parameters both set and one without 

```
execute(): any;
execute(files: string[], filterString: string): any;
```

although I could not find any reference to `filterString` in the actual code
https://github.com/jasmine/jasmine/blob/master/src/core/Env.js#L262

Reply:
I agree with the proposed change.

I thought these were typings definitions for https://github.com/jasmine/jasmine-npm (where both parameters do have an effect), since it says on the [README.md](https://github.com/DefinitelyTyped/DefinitelyTyped) of DefinitelyTyped
>For an NPM package "foo", typings for it will be at "(at)types/foo"

and you install jasmine-npm on npm by typing
```
npm install jasmine
```

See also https://www.npmjs.com/package/jasmine.
This is kind of confusing.

Reply:
ok, this confused me as well 😕 

looking at the correct source code here 
https://github.com/jasmine/jasmine-npm/blob/c3271d21b9fdc72205cd9c712e7a4ca1ebd034fc/lib/jasmine.js#L174

it looks like both parameters are optional, independent from each other so your code is indeed correct 👍

Reply:
So what are these typings for if not for the npm package `jasmine`?

Reply:
@andy-ms The npm package is called `jasmine` so these are the correct typings for that package.
But the repository on Github is called [jasmine-npm](https://github.com/jasmine/jasmine-npm) 

The core mode [jasmine](https://github.com/jasmine/jasmine) is not available on npm at all, therefore the confusion

@Engineer2B linked to the correct repository and the changes are correct, I simply looked in  the wrong repository